ipet/parsing/StatisticReader.py

```diff
@@ -136,6 +136,33 @@ def turnIntoFloat(self, astring):
 ######################################################################################
 # DERIVED Classes
 
+class NodeNameReader(StatisticReader):
+    context = [Key.CONTEXT_LOGFILE, Key.CONTEXT_ERRFILE]
```

Comment:
I don't think it makes sense to have both contexts, as data will only be stored from one of the two files (you only use a single data key).